HotSpot/View Controllers/MainContainerViewController.h

```diff
@@ -8,12 +8,21 @@
 
 #import <UIKit/UIKit.h>
 #import "MapKit/MapKit.h"
+#import <Speech/Speech.h>
 NS_ASSUME_NONNULL_BEGIN
 
-@interface MainContainerViewController : UIViewController <UISearchBarDelegate, MKLocalSearchCompleterDelegate>
+@interface MainContainerViewController : UIViewController <UISearchBarDelegate, MKLocalSearchCompleterDelegate, SFSpeechRecognizerDelegate, UISearchDisplayDelegate> {
+    
+    // affiliated with voice search
+    SFSpeechRecognizer *speechRecognizer;
+    SFSpeechAudioBufferRecognitionRequest *recognitionRequest;
+    SFSpeechRecognitionTask *recognitionTask;
+    AVAudioEngine *audioEngine;
```

Comment:
lines 16-20: why are you doing this?

Reply:
This was removed in the most recent commit